flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -103,37 +119,68 @@ def run(argv=None, save_main_session=True):
     with beam.Pipeline(options=options) as p:
 
         # Read from BigQuery into a PCollection.
-        messages = (
-            p
-            | 'Read from BigQuery Table' >>
-            beam.io.ReadFromBigQuery(
-                table=known_args.input_table
+        if known_args.query is not None:
+            messages = (
+                p
+                | 'Read from BigQuery Table' >>
+                beam.io.ReadFromBigQuery(
+                    query=known_args.query
```

Comment:
I think it might be helpful to add a WARN or INFO log if the query has no LIMIT parameter set, in case user wonders why their pipeline is taking such a long time.  It doesn't look like this code has much logging so will need to think through the effort or if some alternative way to let user know (like maybe when inputs are being parsed)